src/viur/core/module.py

```diff
@@ -43,8 +43,9 @@ def __init__(self, func: t.Callable):
         self.signature = inspect.signature(self._func)
 
         # Guards
-        self.skey = None
+        self.skey = False
         self.access = None
```

Comment:
I think these are obsolete, as the configuration is taken from the closure.

Reply:
We use the `skey` and `access` vars on some ponits  like
https://github.com/viur-framework/viur-core/blob/3c8680fde2c95ba67392486d23cf6209641e7fab/src/viur/core/module.py#L339-L343